### PR TITLE
List AmityCoin (AMIT)

### PR DESCRIPTION
**Pull request by hooftly**
Offical Project URL: https://www.getamitycoin.org

Official Block Explorer: https://explorer.getamitycoin.org/ex2/index.html

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Amitycoin.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class Amitycoin extends Coin {
+
+    public Amitycoin() {
+        super("Amitycoin", "AMIT", new RegexAddressValidator("^amit[1-9A-Za-z^OIl]{94}"));
+    }
+}
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Amitycoin.java]: Make sure to add newlines at the end of each file.

File: assets/src/test/java/bisq/asset/coins/AmitycoinTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        assertInvalidAddress("amitAcVJTUZKJtYYsosMXJBQeEbt3ZV9qSvoQ1EqkvA45MRUaYWECYNKyRZ82BvLM9MPD2Gpud3DbGzGsStKnZ9");
+        assertInvalidAddress("amitAcVJTUZKJtYYsosMXJBQeEbt3ZV9qSvoQ1EqkvA45MRUaYWECYNKyRZ82BvLM9MPD2Gpud3DbGzGsStKnZ9x5yKVPVGJUaljashfeafh");
+        assertInvalidAddress("");
+        assertInvalidAddress("amitAcVJTUZKJtYYsosMXJBQeEbt3ZV9qSvoQ1EqkvA45MRUaYWECY#RoPOWRwpsx1F");
+        assertInvalidAddress("amitAcVJTUZKJtYYsosMXJByRZ82BvLM9MPD2Gpud3DbGzGsStKnZ9x5yKVPVGJUaJbc2q4C4fWN$C4fWNLoDLDvADvpjNYdt3sdRB434UidKXimQQn");
+        assertInvalidAddress("dsfkjasd56yaSDdguaw");
+        assertInvalidAddress("KEKlulzlksadfwe");
+        assertInvalidAddress("HOleeSheetdsdjqwqwpoo3");
+    } 
+}
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/coins/AmitycoinTest.java]: Make sure to add newlines at the end of each file.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK invalid commit message and PR description. Read asset listing guidelines before submitting PRs.

The code looks good. Needs just a codestyle touch.

**Comment by hooftly** [comment]
I will start again apologies.

**Comment by blabno** [comment]
Please reuse this PR, just reopen it and push with force amended commit.

**Comment by hooftly** [comment]
Understood

**Comment by hooftly** [comment]
Let me know if I bungled this up.

**Comment by hooftly** [comment]
Can I close this and try again?  Apologies for the mess

**Comment by blabno** [comment]
Grant me the access to your fork and I'll clean this up.

**Comment by hooftly** [comment]
Access Granted.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**